product_docs/docs/pgd/5/node_management/maintainance_with_proxies.mdx

```diff
@@ -4,57 +4,57 @@ title: Maintenance commands through proxies
 
 ## Maintenance and performance
 
-As a general rule, you should never perform maintenance operations on a cluster's write leader. 
+As a general rule, don't perform maintenance operations on a cluster's write leader. 
 Maintenance operations such as `VACUUM` can be quite disruptive to the smooth running of a busy server and often detrimental to workload performance.
-Therefore it is best to run maintenance commands on any node in a group that isn't the write leader.
-Generally, this requires you connect directly and issue the maintenance commands on the non-write leader nodes.
-But there are situations where this is not possible.
+Therefore, it's best to run maintenance commands on any node in a group that isn't the write leader.
+Generally, this requires you to connect directly and issue the maintenance commands on the non-write-leader nodes.
```

Comment:
This feels imprecise with compared to There are some situations.

Reply:
"There are" is a weak sentence starter (or in this case phrase starter). I don't think it's any less precise. You can revert it if you feel strongly about it.